src/components/Options/OptionsText.vue

```diff
@@ -12,7 +12,7 @@ import NcEmptyContent from '@nextcloud/vue/components/NcEmptyContent'
 import NcActions from '@nextcloud/vue/components/NcActions'
 import NcActionButton from '@nextcloud/vue/components/NcActionButton'
 
-import DeleteIcon from 'vue-material-design-icons/Delete.vue'
+import DeleteIcon from 'vue-material-design-icons/DeleteOutline.vue'
```

Comment:
would be `TrashCanOutline` since we decided to go for Material Symbols, and from the Material Design Icons that would be the matching icon